packages/itinerary-body/src/TransitLegBody/index.tsx

```diff
@@ -118,8 +120,16 @@ class TransitLegBody extends Component<Props, State> {
         ? transitOperator.logo
         : agencyBrandingUrl;
 
-    const expandAlerts =
-      alertsExpanded || (leg.alerts && leg.alerts.length < 3);
+    const maximumAlertCountToShowUncollapsed = 2;
+    const shouldShowAlertsToggle =
+      leg.alerts &&
+      (leg.alerts.length > maximumAlertCountToShowUncollapsed ||
```

Comment:
```suggestion
      (leg.alerts?.length > maximumAlertCountToShowUncollapsed ||
```

You should be able to write this. It will be compiled down to more browser-compatible, verbose code upon packaging.

Reply:
There is a second instance of the same refactoring a few lines below.

Reply:
Thanks! I pulled this out, and while I was in there I updated the booleans to remove some duplicated logic I created (`leg.alerts.length > maximumAlertCountToShowUncollapsed` was used in two spots).